src/main/java/com/talentpath/airTNB/controllers/ListingController.java

```diff
@@ -31,4 +31,10 @@ public Listing getListingById(@PathVariable Integer id) throws NullListingExcept
     public Listing addListing(@RequestBody Listing toAdd){
         return listingService.addListing(toAdd);
     }
+
+    @GetMapping("/getCloseListingsByCityAndState/{listingId}")
+    public List<Listing> getByCityAndState(@PathVariable Integer listingId) throws NullListingException {
+        System.out.println("HEY DOUBLE CHECKING");
+        return listingService.getListingsByCityAndState(listingId);
```

Comment:
Likely want to refactor at this level to parse city and state, instead of sending the method an ID.